docroot/themes/custom/uids_base/scss/components/card.scss

```diff
@@ -352,6 +352,17 @@
   margin: 0 0 .125rem
 }
 
+// Remove margin-bottom for last item in card--lists.
+.views-row:last-child .card--list {
+  margin-bottom: 0;
+}
+
+.block-inline-blockuiowa-events,
+.view-article-list-block {
+  .card--list:last-child  {
+    margin-bottom: 0;
+  }
+}
```

Comment:
I don't love putting these here because they seem more relevant to views than the card.

Reply:
Good point, I moved these to `views.scss` for now.

Reply:
Thank you!